Extension/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.5.0-insiders: June 14, 2021
+### Enhancements
+* Add "Symbol Options" for CppVsdbg to configure symbol settings [PR #7680](https://github.com/microsoft/vscode-cpptools/pull/7680)
+* Update CppVsdbg to use newer CppEE and msdia. 
+
+### Bug Fixes
+* Fix Find All References reporting certain references in headers as inactive. [#7609](https://github.com/microsoft/vscode-cpptools/issues/7609)
+* Fix IntelliSense process crash and tag parser failure with columns > 65535. [#7621](https://github.com/microsoft/vscode-cpptools/issues/7621)
+* Fix incorrect translations.
+  * jogo- (@jogo-) [PR #7625](https://github.com/microsoft/vscode-cpptools/pull/7625)
+* Fix `autocompleteAddParentheses` for some template argument deduction cases. [#7626](https://github.com/microsoft/vscode-cpptools/issues/7626)
+* Fix some incorrect IntelliSense errors. [#6639](https://github.com/microsoft/vscode-cpptools/issues/6639], [#7630](https://github.com/microsoft/vscode-cpptools/issues/7630)
+* Change references to OS X to macOS.
```

Comment:
There should be an "of" --> "Change references of "OS X" to macOS."

Note: already updated this in internal copy on OneNote.